### PR TITLE
Add QtPushButton

### PR DESCRIPTION
**Pull request by Mimoja**

### CODE REVIEW

File: src/QtWidgets/QtPushButton.h [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+#include <QPushButton>
+
+@interface QtPushButton: QtAbstractButton
+@property (readonly) QPushButton* qPushButton;
+@property QMenu* menu;
+@property bool autoDefault;
+@property (getter=isDefault, setter=setDefault:) bool default_;
+@property (getter=isFlat) bool flat;
+
+
```
Comment (Midar) [inline src/QtWidgets/QtPushButton.h]: Remove this extra empty line

// Edit: Clarification: It's currently 2 empty lines. Make it one empty line.

File: src/QtWidgets/QtPushButton.h [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- initWithQPushButton: (QPushButton*)qPushButton;
+- initWithText: (OFString*)text;
+- initWithIcon: (QIcon)icon
+	  text: (OFString*)text;
+
```
Comment (Midar) [inline src/QtWidgets/QtPushButton.h]: Remove this extra empty line

File: src/QtWidgets/QtPushButton.h [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+
+#import "QtAbstractButton.h"
+
+#include <QMenu>
+#include <QPushButton>
+
+@interface QtPushButton: QtAbstractButton
+@property (readonly) QPushButton* qPushButton;
```
Comment (Midar) [inline src/QtWidgets/QtPushButton.h]: `QPushButton *qPushButton`

**Review by Midar — CHANGES_REQUESTED** [review]
Thanks for the patch, just 3 very minor nits. Overall it is looking very good!

**Comment by Mimoja** [comment]
done

**Pull request merged**